CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.6.2] - 2024-09-30
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.6.1...v1.6.2)
```

Comment:
@jpadilla could we remove line 19? It is slightly misleading; we didn't change the ReadChanges API, we changed the storage interface, which most users do not care about.